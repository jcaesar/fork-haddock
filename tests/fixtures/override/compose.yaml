services:
  myservice:
    image: redis
    command: python app.py
    expose:
      - "3000"
    environment:
      - FOO=original
      - BAR=original
    labels:
      - FOO=original
      - BAR=original
    volumes:
      - ./original:/foo
      - ./original:/bar
    devices:
      - /original:/foo
      - /original:/bar
    configs:
      - source: foo
        target: original
      - source: bar
        target: original
