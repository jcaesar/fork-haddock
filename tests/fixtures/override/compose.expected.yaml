services:
  myservice:
    image: flask
    command: python otherapp.py
    expose:
      - '3000'
      - '4000'
      - '5000'
    environment:
      - FOO=original
      - BAR
      - BAZ=local
    labels:
      - FOO=original
      - BAR=local
      - BAZ=local
    volumes:
      - ./original:/foo
      - ./local:/bar
      - ./local:/baz
    devices:
      - /original:/foo
      - /local:/bar
      - /local:/baz
    configs:
      - source: foo
        target: original
      - source: bar
        target: local
      - source: baz
        target: local
